server_admin/topics/clients/oidc/con-confidential-client-credentials.adoc

```diff
@@ -0,0 +1,78 @@
+[id="con-confidential-client-credentials_{context}"]
+
+[[_client-credentials]]
+
+==== Confidential Client Credentials
+
+If the <<_access-type, access type>> of the client is set to `confidential`, the credentials of the client must be configured under the `Credentials` tab.
+
+.Credentials Tab
+image:{project_images}/client-credentials.png[Credentials Tab]
+
+The `Client Authenticator` drop-down list specifies the type of credential to use for your client. 
+
+*Client ID and Secret*
+
+This is the default setting. The secret is automatically generated for you and the `Regenerate Secret` button recreates the secret if necessary.
+
+*Signed JWT*
```

Comment:
You had mentioned that you removed a screen shot that looks out of date because it does not show  "Signature Algorithm."  Unfortunately, I only see that field when you create a SAML app. Can you clarify?

Reply:
After digging further, it looks like the screen depends on the client that is being used. For example, I am using the default clients that are in my my Keycloak environment right now. The "broker" client shows a "signature algorithm" field in the "credentials" section but another client, "account", shows different credentials altogether. Can show you when we next sync up.